scripts/server_benchmark.py

```diff
@@ -11,17 +11,20 @@
 import argparse
 import cProfile
 
-import ga4gh.backend as backend
-import ga4gh.protocol as protocol
-import ga4gh.datarepo as datarepo
-
 import guppy
 
+import utils
+utils.ga4ghImportGlue()
+import ga4gh.backend as backend  # noqa
+import ga4gh.protocol as protocol  # noqa
+import ga4gh.datarepo as datarepo  # noqa
+
```

Comment:
It's because of the ``utils.ga4ghImportGlue()`` call. Pep8 doesn't want you to put in imports after code.